### DISCUSSION TITLE
Importing many to many objects related field through Data Object Importer

### DISCUSSION THREAD

**Discussion by nirbanc2020** [pimcore/pimcore#11390] [General]
Hi, I imported many fields like input types,many to one relation,images,gallery through asset type data source in csv format.But how to import many to many object related fields like category ?In Import Settings tab and in Mappings section Data Target, proper field name is not comming to import.I also tried to import by creating array for many to many object related field in csv but still it was not possible.
![data-importer](https://user-images.githubusercontent.com/73533273/152916438-f133d8b4-a45b-45cf-98bb-8b2b71e28cc5.png)
![data-importer](https://user-images.githubusercontent.com/73533273/152916495-087f2bf6-1863-4de2-94b2-e60d5220ce60.png)

**Comment by idumaire** [comment]
hi,

CSV import was more simple ! 

In this case you have to add 2 transformation items :
- explode
- Load DataObject based on Path for example to use the fullpath reference element

Then select Type : Many to many relation and you will have your attribute in Field Name.

@PIMcore Team : please add more documentation to load standard field like "country" ....
